ui/components/app/tab-bar/tab-bar.js

```diff
@@ -3,20 +3,39 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
 import { Icon, IconName, IconSize } from '../../component-library';
+import Box from '../../ui/box';
+import {
+  BorderRadius,
+  Color,
+  DISPLAY,
+} from '../../../helpers/constants/design-system';
 
 const TabBar = (props) => {
   const { tabs = [], onSelect, isActive } = props;
 
   return (
     <div className="tab-bar">
       {tabs.map(({ key, content, icon }) => (
-        <button
+        <Box
+          as="button"
```

Comment:
Can this just be `<Button>`?  I think the new one is based on `Box`

Reply:
Hmm! I didn't use the New Button component because of these reasons: 
- I will have to override its css since I don't need the Button Css here.
- This is a Tab and I thought of implementing it the same way we did for MultichainTokenListItem and AccountList.